### DISCUSSION TITLE
Is it possible to have two sets of release notes for a single dependency?

### DISCUSSION THREAD

**Discussion by jvmdc** [renovatebot/renovate#29020] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket Server - Renovate 37.342.1

### Please tell us more about your question or problem

We're using Renovate to bump our Renovate Helm chart. The PR does include the release notes as defined here: https://github.com/renovatebot/helm-charts/releases, however in most cases the interesting stuff is going on here:  https://github.com/renovatebot/renovate/releases

While I suppose I could override it using `customChangelogUrl`, I assume this would make me lose the original release notes.

Is it possible to have two sets of release notes for a single dependency (Renovate Helm chart in this case) or is there another way to obtain having access to both sets of release notes directly in the PR?

**Comment by rarkins** [comment]
There is no feature for this
